.github/workflows/create_trello_card_template.yaml

```diff
@@ -0,0 +1,26 @@
+name: Create Trello card
+
+on:
+  workflow_call:
+    secrets:
+      TRELLO_API_KEY:
+        required: true
+      TRELLO_API_TOKEN:
+        required: true
+
+jobs:
+  create-card:
+    runs-on: ubuntu-latest
+    steps:
+      - name: echo ref_name
+        run: echo GITHUB.HEAD_REF_NAME ${GITHUB_HEAD_REF_NAME}
+      - name: create a card
+        uses: nentgroup/pr-trello-action@v2
+        with:
+          api-key: '${{ secrets.TRELLO_API_KEY }}'
+          api-token: '${{ secrets.TRELLO_API_TOKEN }}'
+          list-id: '63ce5064a4859b0387f46b9b'
```

Comment:
What column does this put it in, is it a specific dashboards to watch style one?

Reply:
That did cross my mind, but I've just gone with In review so far.

Reply:
Fair - in that case similar to below, is that worth a comment next to it for future reference?

Reply:
Yep, added as a comment. Also this is why I've added it as a template, so that we can easily switch it over to another column (or add other labels) for all dashboards using it if that's ever required.